src/main/java/seedu/medibook/model/util/SampleDataUtil.java

```diff
@@ -72,4 +78,43 @@ public static ReadOnlyMediBook getSampleMediBook() {
                 .collect(Collectors.toSet());
     }
 
+    private static Patient[] generateRecords(Patient[] patients) {
+        for (Patient patient : patients) {
+            LocalDate date = LocalDate.now();
+            String currentHeight = patient.getHeight().get().value;
+            String currentWeight = patient.getWeight().get().value;
+            Record patientRecord = patient.getRecord();
+            double weightVariance = Math.random() * 10;
+            for (int i = 0; i < Math.random() * 20; i++) {
+                String dateString = toDateString(date);
+                Date recordDate = new Date(dateString, true);
+                Height recordHeight = new Height(currentHeight);
+                Weight recordWeight = new Weight(currentWeight);
+                patientRecord.addHeightRecord(recordDate, recordHeight);
+                patientRecord.addWeightRecord(recordDate, recordWeight);
+
+                date = date.minusDays((int) (Math.random() * 20));
+                int updatedHeight = Integer.parseInt(currentHeight) - (int) (Math.random() * 1.5);
+                currentHeight = Integer.toString(updatedHeight);
+
+                double updatedWeight = Double.parseDouble(currentWeight) + (weightVariance * (Math.random() - 0.5));
+                currentWeight = toOneDecimal(Double.toString(updatedWeight));
+            }
+        }
+        return patients;
+    }
```

Comment:
This is just to create sample records for testing?

Reply:
Yep